### DISCUSSION TITLE
Limiting large downloads / updates to Wi-Fi

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#461] [Q&A] [answered]
Once I had the problem to waste my mobile data quota while downloading/updating some larger maps not knowing that my WiFi connection was down. Is it possible to offer a setting for large downloads/updates of maps, POIs and routing data? WiFi only or mobile data and WiFi? I don't want to forbid mobile data to the Cruiser app completely because sometimes I use the online search and OpenRouteService. I just want to prevent big downloads via mobile data connection.
In Google Maps there is something similar for offline maps available.

**Comment by devemux86** [comment]
@kds112 thanks for the suggestion.

We can add a similar setting for the downloads.

**Reply by bios4** [reply]
Or just a simple message (similar to Google Play): "Do you really want to download XY using mobile data", with options "Confirm" and "Cancel"...

**Comment by devemux86** [comment]
<img src="https://github.com/devemux86/cruiser/assets/3484020/b2806d51-f9ef-4417-a449-69728e2b960c" width="50%"/>

**Reply by kds112** [reply]
Great!! 🏁

**Reply by kds112** [reply]
3.0.41 works very well!  Thanks a lot for this feature 👍

**Comment by kds112** [comment]
Sorry there is still one minor issue... The setting is profile-dependend. It should be a global setting.
I thought all advanced settings are global, aren't they? 😳

**Reply by kds112** [reply]
Maybe, all global settings should be named with "(global)"?

**Reply by devemux86** [reply]
> It should be a global setting.

@kds112 I can change it.

> I thought all advanced settings are global, aren't they?

Only a few settings are global:
- App language
- Download preferences
- Storage folder
- Extended settings
- Extended menu

**Reply by kds112** [reply]
Maybe you can name them as "(global)" thus everybody knows it at first glance? 🤔

**Reply by devemux86** [reply]
We could, although being an advanced function, several users will wonder what "(global)" means.

**Reply by kds112** [reply]
Then maybe "(global setting)"? 🙃 Or you create a subfolder in application settings called "Global setting" and move them into it...?

**Reply by bios4** [reply]
Don't complicate things, please... 😉

While I agree that the new setting should be stored independently from the profile, I don't see a benefit of arbritrarily moving around/renaming options and settings.

**Comment by kds112** [comment]
Thanks for making this parameter as a global setting in the latest beta 👍

**Comment by devemux86** [comment, marked as answer]
Download setting for Wi-Fi or mobile network is available in [Cruiser 3.0.41](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
